x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel.tsx

```diff
@@ -222,7 +222,7 @@ export function InnerWorkspacePanel({
                 />
               </EuiLink>
             </small>
-          </p>
+          </div>
```

Comment:
After:

<img width="487" alt="Screenshot 2020-07-15 17 09 25" src="https://user-images.githubusercontent.com/666475/87609240-f2aa5300-c6cf-11ea-9a95-5ea13c3e6c76.png">

Reply:
It fixes this warning:

```
console.error
      Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.
          in div (created by EuiIcon)
          in EuiIcon (created by Context.Consumer)
          in EuiI18n (created by EuiLink)
          in a (created by EuiLink)
          in EuiLink (created by InnerWorkspacePanel)
          in small (created by InnerWorkspacePanel)
          in p (created by InnerWorkspacePanel)
```

Reply:
Those all stem from `EuiIcon` and it'll be hard to get rid of them until we change the element that `EuiIcon` renders. I'd rather keep the appropriate text elements for screen-readers when possible and I'll make an issue for this in EUI.

Reply:
Ok this is weird, **EuiIcon** does **not** in fact ever render a `<div>` (I think it did way back when, but now it's just a simple `<svg>`). So this error message is actually completely false. Do we know who setup this DOM validator so I can follow up?

Reply:
Ok, so I think I've tracked this down to the fact that in Kibana, because EuiIcon is dynamically imported, jest tests mock the component into rendering a `<div>`. So the React warning is actually remarking on the snapshots and not the **actual** dom. I'll see if the mocks are also coming from EUI, and if so, just get them to render a `span`.

But in the meantime, can you revert this `p` to `div` change? If you truly like the minimized space between the two lines, then wrap them all in a single `<p>` with a `<br />` in between.